### ISSUE TITLE
JSDoc: Comments are missing

### ISSUE THREAD

**Issue by reicheltp** [dotansimha/graphql-code-generator#4110]
**Describe the bug**
Comments are not copied to output with plugin JSDoc.

**To Reproduce**

1. My GraphQL schema:

```graphql
type DemoPayload {
  """
  Test 123
  """
  test: String!
}
```

2. My `codegen.yml` config file:

```yml
generates:
  schema.js:
    plugins:
      - jsdoc
```

3. False Result

```js
/**
 * @typedef {Object} DemoPayload
 * @property {string} test
 */
```

**Expected behavior**

Comments are copied to the schema.js

```js
/**
 * @typedef {Object} DemoPayload
 * @property {string} test Test 123
 */
```

**Environment:**

Used: https://graphql-code-generator.com/

- OS: 
- `@graphql-codegen/...`: 
- NodeJS: 

**Additional context**

**Comment by ardatan** [comment]
Could you create a reproduction using the template below?
https://codesandbox.io/s/github/dotansimha/graphql-code-generator-issue-sandbox-template

**Comment by reicheltp** [comment]
@ardatan Done: https://codesandbox.io/s/keen-star-crfuq?file=/schema.js

Look at `schema.js` to see that the comments are missing.

**Comment by dotansimha** [comment]
Fixed in: https://github.com/dotansimha/graphql-code-generator/pull/4142

**Comment by dotansimha** [comment]
Fixed in `v1.15.2`
